### PR TITLE
updated notebook intro

### PR DESCRIPTION
**Pull request by ashwinkumar12345**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/notebooks/index.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ A Kibana notebook is an interface that lets you easily combine live visualizatio
 
 With Kibana notebooks, you can interactively explore data by running different visualizations and share your work with team members to collaborate on a project.
 
-You can add visualization paragraphs for creating graphs and markdown paragraphs for writing descriptions. You can choose multiple timelines to compare and contrast visualizations.
+A notebook is a document made up of two types of cells or paragraphs. You can add visualizations with a visualization paragraph and text with a markdown paragraph. You also can choose multiple timelines to compare and contrast visualizations.
```
Comment (aetter) [inline docs/notebooks/index.md:14]: I don't think this is much of an improvement. Surely no one knows what a "cell" means in this context. Why not something like this?

"A notebook is a document composed of two elements: Kibana visualizations and paragraphs (Markdown). Choose multiple timelines to compare and contrast visualizations."

Comment (vrphanse) [reply docs/notebooks/index.md:14]: I like it

Comment (vrphanse) [reply docs/notebooks/index.md:14]: How about: A notebook is a document composed of paragraphs. You can add visualizations with a visualization paragraph and text with a markdown paragraph. You also can choose multiple timelines to compare and contrast visualizations. @aetter ?

Comment (aetter) [reply docs/notebooks/index.md:14]: Why do we feel the need to call them "visualization paragraphs?" I can't imagine that making sense to anyone.

Comment (vrphanse) [reply docs/notebooks/index.md:14]: The basic component here is paragraph. And paragraphs essentially are of two types- one for visualization and the other one for markdown.

Comment (vrphanse) [reply docs/notebooks/index.md:14]: Do let me know the best way to word this

Comment (aetter) [reply docs/notebooks/index.md:14]: I think we just delete everything but the last sentence and move "You can also choose multiple timelines to compare and contrast visualizations." onto the previous paragraph. The purpose of the introduction is to answer the question of "should I care about this?" for the reader, and leading off with convoluted terms that we invented doesn't get readers closer to an answer.

Comment (vrphanse) [reply docs/notebooks/index.md:14]: Sure. Thanks a lot. @ashwinkumar12345 Please add and the doc is good to go.

**Pull request merged**